src/palace/manager/sqlalchemy/model/identifier.py

```diff
@@ -811,9 +853,10 @@ def add_link(
         elif media_type and not resource.representation:
             # We know the type of the resource, so make a
             # Representation for it.
-            resource.representation, is_new = get_one_or_create(
+            rep, is_new = get_one_or_create(
                 _db, Representation, url=resource.url, media_type=media_type
             )
+            resource.representation = rep
```

Comment:
Mypy didn't like putting this into a unpacked tuple, putting it on its own line was fine. Just some mypy weirdness, but not a big change so I think its fine.

Reply:
Even though I must've seen it in the code at some point, I had no conscious idea that `resource.representation, is_new = get_one_or_create(...` was even allowed in Python. The more you know.